pkgs/development/libraries/sdbus-cpp/default.nix

```
@@ -34,8 +35,10 @@ let
         systemdLibs
       ];
 
+      inherit codegenFlag;
+
       cmakeFlags = [
-        (lib.cmakeBool "BUILD_CODE_GEN" true)
+        (lib.cmakeBool codegenFlag true)
```

```suggestion
        (lib.cmakeBool (if lib.versionOlder finalAttrs.version "2.0.0" then "BUILD_CODE_GEN" else "SDBUSCPP_BUILD_CODEGEN") true)
```

Making this dependent on `version` should be more robust in overrides, and make this less of a pain to extend / clean up once we finally remove old `sdbus-cpp`.